src/main/java/io/codelex/flightplanner/adminapi/AdminService.java

```diff
@@ -0,0 +1,32 @@
+package io.codelex.flightplanner.adminapi;
+
+import io.codelex.flightplanner.domain.Flight;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AdminService {
```

Comment:
Pa lielam kods izskatās tīri labs, bet man īsti nepatīk tas dalījums Admin/Customer/Test. Tas ir ok, kad dalam controller, bet pēc controller būtu jādala pēc domain nevis pēc lietotāja veida. Mūsu gadījumā domain ir Flight, teorētiski varētu atdlīt arī Airport, bet, manuprāt, nav vērts to darīt. 
Šobrīd Tava shēma ir šāda:
```
AdminController       => AdminService        => AdminInMemoryRepository       \
CustomerController    => CustomerService     => CustomerInMemoryRepository      => Server
TestingController     => TestingService      => TestingInMemoryRepository     /
```
Bet vajadzētu šādi (visi trīs izmanto FlightService):
```
AdminController       \  
CustomerController      =>  FlightService => FlightInMemoryRepository
TestingController     /  
```
Un tad šajā brīdī arī zudīs tās Server klases jēga, jo viss kopīgais var glabāties iekš FlightInMemoryRepository. Un to kas ir atsevišķ iekš AdminService, CustomerService un TestService apvieno vienā FlightService un tas pats ar repository.

Ceru, ka domu uzķēri, bet, ja kas - raksti!

Reply:
Tātad, cik saprotu, tad Controlleri katram ir savi, bet, sākot ar Service, viss iet kopējs?
Ok. Izlabošu.